### ISSUE TITLE
Add support for Clear/Transparent Blur background

### ISSUE THREAD

**Issue by henriquecfreitas** [Kureev/react-native-blur#602]
The `blurType` options supported for iOS does not provide any option for handling a clear background to the `BlurView` component.

That would be useful in many cases, like described on issues #465 , #474 , #407 and others.

The app I'm currently working on is using the `BlurView` component for the Header background in some specific pages, where a `FlatList` component should be able to slightly show the cards content underneath the page Header when scrolling. The problem is that the color displayed on the Header when there is no content behind it (eg: list scroll is on top) is not matching the same color of the Header displayed on pages where it does not use the BlurView. Of course they doesn't need to perfectly match the same outputed RGB, since one of them has a Blur effect applied, but it would be great if they could have at least a similiar feel.

Example:
![image](https://github.com/Kureev/react-native-blur/assets/22528445/88d22b01-8d7e-4d8f-a2bf-f0a733ed267a)

This issue would problably be fixed with the solution proposed by @KirillGudkov on the PR #513

**Comment by shav95** [comment]
I agree with  @henriquecfreitas , I also have a project were i need to put a clear blur view , but I cannot . on android it works but on ios there is no way. 
Dear team please if it is possible find a solution for this issue.

**Comment by haroutunyan** [comment]
Same problem. Please help.

**Comment by shav95** [comment]
@Kureev

**Comment by ugar0ff** [comment]
Same problem

**Comment by w3company-leo** [comment]
same

**Comment by NorseGaud** [comment]
Bump

**Comment by GrishovAleksei** [comment]
up

**Comment by OmarMach** [comment]
up
